tests/drum/test_units.py

```diff
@@ -1605,67 +1592,3 @@ def test_get_default_parameter_values(model_metadata, default_value):
         assert get_default_parameter_values(model_metadata) == {"param_name": default_value}
     else:
         assert not get_default_parameter_values(model_metadata)
-
-
-@pytest.mark.parametrize(
-    "has_hyper_param_in_model_metadata, model_metadata_name, default_parameters",
-    [
-        (
-            True,
-            "model-metadata.yaml",
-            {"penalty": "l2", "dual": 0, "tol": 0.0001, "solver": "lbfgs"},
-        ),
-        (False, "model-metadata-no-hyperparameter.yaml", {}),
-        (False, "no", {}),
-    ],
-)
-def test_cmrunner_init_default_parameter_values(
```

Comment:
this test ensured model-metadata hyperparam default values propagated through mlpiper. No longer needed